tools/ZAPD/ZAPD/ZAPD.vcxproj

```
@@ -305,6 +305,9 @@ python3 ZAPD/genbuildinfo.py</Command>
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup>
+    <None Include="packages.config" />
+  </ItemGroup>
```

This looks like something went wrong in the merge
```suggestion
  <ItemGroup>
    <None Include="packages.config" />
  </ItemGroup>
```